src/ts/components/seekbar.ts

```diff
@@ -408,7 +407,18 @@ export class SeekBar extends Component<SeekBarConfig> {
     if (this.config.smoothPlaybackPositionUpdateIntervalMs !== SeekBar.SMOOTH_PLAYBACK_POSITION_UPDATE_DISABLED) {
       this.configureSmoothPlaybackPositionUpdater(player, uimanager);
     }
-    this.configureMarkers(player, uimanager);
+
+    // Initialize markers
+    this.initializeTimelineMarkers(player, uimanager);
+  }
+
+  private initializeTimelineMarkers(player: PlayerAPI, uimanager: UIInstanceManager): void {
+    const timelineMarkerConfig = {
+      cssPrefix: this.config.cssPrefix,
+      snappingRange: this.config.snappingRange,
+    };
+    this.timelineMarkersHandler = new TimelineMarkersHandler(timelineMarkerConfig, () => this.seekBar.width(), this.seekBarMarkersContainer);
+    this.timelineMarkersHandler.configure(player, uimanager);
```

Comment:
You pass the `player` and `uiManager` now in the constructor.

Reply:
Trying to keep the number of parameters to maximum 3, or are we OK with passing more. Feel like having `configure` seems ok here to avoid this problem.

Reply:
I think we are ok with passing more. However fine 4 me but then please rename the method. `configure` is our 'configure' method for real components and one might get confused here. Maybe rename to `setup` or `initialize` but still I would move them into the constructor.

Reply:
https://github.com/bitmovin/bitmovin-player-ui/pull/310/commits/55763804214bfb6e2faa0ade8a2ce3455b51d5b8